spinnyBoi.py

```diff
@@ -55,9 +59,13 @@ async def on_message(self, message):
         for key in reaction_dict.keys():
             if str(message.content).lower().find(key) != -1:
                 try:
-                    await message.add_reaction(reaction_dict[key])
+                    if len(reaction_dict[key]) == 1:
```

Comment:
this if statement isn't necessary, lets just grab a random one from the array even if there's only a single element

Reply:
I was thinking about that too, makes sense, done.